bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -314,11 +369,50 @@ func (p *packer) encodeTuple(t Tuple, nested bool) {
 // call Pack when using a Tuple with a FoundationDB API function that requires a
 // key.
 func (t Tuple) Pack() []byte {
```

Comment:
I think the godoc description needs to be updated here with an update to say that one can give it a `Versionstamp` in addition to the previous types. It should also specify that this will panic if there is an incomplete `Versionstamp` in the Tuple.

Reply:
👍

Reply:
It looks like the thing about panicking has been resolved (though I'm not sure it's true...), but the list of tuple-able types is still missing `Versionstamp`.